electron/app/index.tsx

```diff
@@ -17,11 +15,8 @@ export default function Root() {
   useEffect(() => {
     async function createInitialLoad() {
       try {
-        await initChatFiles();
-        const chatDB = initializeDB(appChatDBDirectoryPath);
-        await dropAllTables(chatDB);
-        await createAllTables(chatDB);
-        setDB(chatDB);
```

Comment:
basically moved everything to here: `coreInit()`... eventually we'll have to break it out and prompt